dockers/docker-dhcp-server/cli-plugin-tests/test_show_dhcp_server.py

```diff
@@ -110,3 +110,28 @@ def test_show_dhcp_server_ipv4_range_single_ip(self, mock_db):
         assert result.exit_code == 0, "exit code: {}, Exception: {}, Traceback: {}".format(result.exit_code, result.exception, result.exc_info)
         assert result.stdout == expected_stdout
 
+    def test_show_dhcp_server_ipv4_option_without_name(self, mock_db):
+        expected_stdout = """\
+Option Name      Option  Value        Type
```

Comment:
Would it better to change second column name to "Options ID"?

Reply:
@yaqiangz  updated